File: AppPadaria/src/app/Pedido.ts
export class Pedido {
    // armazena o id do pedido
    id: number = 0;
    // armazena a data e hor[ario do pedido
    datapedido: Date | undefined;
    // armazena o nome do cliente que fez o pedido
    nomepedido: string = '';
    // armazena o status atual do pedido
    status: string = '';
    // armazena o id do cliente que fez o pedido
    clienteid: number = 0;
    }

File: trabalho/Models/Pedido.cs
///CHATGPT
using System.ComponentModel.DataAnnotations;

namespace API_Padaria.Models
{
    public class Pedido
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public DateTime DataPedido { get; set; }

        [Required]
        public string? NomePedido { get; set; }
        public string? Status { get; set; }

        public int ClienteId { get; set; }
    }
}